model/github_runner.rb

```diff
@@ -34,6 +34,13 @@ def display_state
     end
   end
 
+  def log_duration(message, duration)
+    values = {ubid: ubid, label: label, repository_name: repository_name, duration: duration, vm_ubid: vm.ubid, arch: vm.arch, cores: vm.cores}
+    values[:vm_host_ubid] = vm.vm_host.ubid if vm.vm_host
+    values[:pool_ubid] = VmPool[vm.pool_id].ubid if vm.pool_id
+    Clog.emit(message) { {message.tr(" ", "_") => values} }
```

Comment:
Why do we do replacement here instead of sending message with underscores in the first place?